pydantic/validate_call.py

```diff
@@ -32,7 +34,7 @@ def validate_call(
     validate_return: bool = False,
 ) -> AnyCallableT | Callable[[AnyCallableT], AnyCallableT]:
     """
-    Decorator to validate the arguments passed to a function and, optionally, the return value.
+    Returns a decorated version of the function that validates the arguments and, optionally, the return value.
```

Comment:
To me, if a function description starts with "returns", then it shouldn't be here, but on the "Return" part of the docstrings. 👀